R/Belgium.R

```diff
@@ -0,0 +1,208 @@
+#' Belgium Class for downloading, cleaning and processing notification data
+#'
+#' @description Country specific information for downloading, cleaning
+#'  and processing COVID-19 region level 1 and 2 data for Belgium.
+#'
+#' @details Inherits from `DataClass`
+#' @source https://epistat.sciensano.be/Data/COVID19BE_CASES_AGESEX.csv
+#' @source https://epistat.sciensano.be/Data/COVID19BE_HOSP.csv
+#' @source https://epistat.sciensano.be/Data/COVID19BE_MORT.csv
+#' @export
+#' @examples
+#' \dontrun{
+#' region <- Belgium$new(verbose = TRUE, steps = TRUE, get = TRUE, level = "2")
+#' region$return()
+#' }
+Belgium <- R6::R6Class("Belgium",
+  inherit = DataClass,
+  public = list(
+
+    # Core Attributes
+    #' @field country name of country to fetch data for
+    country = "Belgium",
+    #' @field supported_levels A list of supported levels.
+    supported_levels = list("1", "2"),
+    #' @field supported_region_names A list of region names in order of level.
+    supported_region_names = list("1" = "region",
+                                  "2" = "province"),
+    #' @field supported_region_codes A list of region codes in order of level.
+    supported_region_codes = list("1" = "iso_3166_2_region",
+                                  "2" = "iso_3166_2_province"),
+    #' @field common_data_urls List of named links to raw data that are common
+    #' across levels. The first entry should be named main.
+    common_data_urls = list(
```

Comment:
remove  "The first entry should be named main." as this is more direction for the author adapting another class